.github/workflows/tests.yml

```
@@ -51,8 +51,8 @@ jobs:
     - name: Create test environment file
       run: |
         cp .env.example .env
-        echo "FLASK_SECRET_KEY=test-secret-key" >> .env
-        echo "FLASK_DEBUG=False" >> .env
+  echo "FASTAPI_SECRET_KEY=test-secret-key" >> .env
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 54: could not find expected ':'

(syntax-check)